wpilibc/src/main/native/cpp/simulation/ElevatorSim.cpp

```diff
@@ -13,17 +13,14 @@ using namespace frc;
 using namespace frc::sim;
 
 ElevatorSim::ElevatorSim(const LinearSystem<2, 1, 1>& plant,
-                         const DCMotor& gearbox, double gearing,
```

Comment:
Why was gearing removed from this constructor? This is a breaking change. Both physical constants and feedforward constants should be supported.

Reply:
It's not needed.  The only place that gearing is used is in the LinearSystem::Elevator constructor at line 34.  

It was also used in ElevatorSim::GetCurrentDraw method at line 82, but it can be determined from the inherited LinearSystem m_plant.

Reply:
Would it be better to put it back in and have it set as a field?  Then bring it back to GetCurrent? Then in the feedforward constructor have those parameters determine the gearing instead?

Reply:
You're right, the linear system object already includes gearing. The DCMotor and gearing was only used for current calculations, but the new approach avoids them.